ext_libs/ext_libs.cmake

```diff
@@ -0,0 +1,51 @@
+# Use position independent code for all external libraries
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+if(FMT_SYS_DEP)
+  # fmt was initially built against 7.1.3. Its possible earlier versions will also work, but that needs to be tested
+  find_package(fmt 7.1.3 REQUIRED)
```

Comment:
Does that mean exactly 7.1.3 or anything newer?

Reply:
That should be >= 7.1.3. The comment under rapidjson explains